Bio/GenBank/__init__.py

```diff
@@ -482,18 +111,12 @@ def __iter__(self):
         return iter(self.__next__, None)
 
 
-class ParserFailureError(Exception):
+class ParserFailureError(ValueError):
     """Failure caused by some kind of problem in the parser."""
 
     pass
 
 
-class LocationParserError(Exception):
```

Comment:
Removing this, or rather moving it under ``Bio/SeqFeature.py`` is logical, but could break some people's scripts.

Can we leave a stub here too? e.g. ``from Bio.SeqFeature import LocationParserError`` with a comment about it being for backward compatibility?

Reply:
It's already there, as `LocationParserError` is still needed in `Bio/GenBank/__init__.py`.